pkg/controller/cassandracluster/generator.go

```diff
@@ -395,7 +401,7 @@ func generateCassandraStatefulSet(cc *api.CassandraCluster, status *api.Cassandr
 	// Merge cassandra main container environment variables into sidecars.
 	for idx, container := range ss.Spec.Template.Spec.Containers {
 		if container.Name != cassandraContainerName {
-			ss.Spec.Template.Spec.Containers[idx].Env = append(ss.Spec.Template.Spec.Containers[idx].Env, cassandraContainer.Env...)
+			ss.Spec.Template.Spec.Containers[idx].Env = append(container.Env, bootstrapContainer.Env...)
```

Comment:
@erdrix , do you think we really need it ? We could probably remove some of them and maybe push only DC, RACK and CLUSTER_NAME ? why would they need the heap size, the seeds etc.. ?

Reply:
When I did it, I took the side of pushing them all because we had a low visibility on sidecar usage and therefore the env vars required for these ... So if the necessary are the ones you specified no problems for me :)